Create opc-ua-connector.md

delion-tholens opened the pull request.

delion-tholens committed:
Create opc-ua-connector.md

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/mendix/docs?pullRequest=7664) <br/>All committers have signed the CLA.

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

Added docs for more actions.
Most of it is copied at this moment from the microflows

delion-tholens committed:
Update opc-ua-connector.md

Finished first version of the actions parts

delion-tholens committed:
Update opc-ua-connector.md

Added steps to set up your connection

delion-tholens committed:
Update opc-ua-connector.md

Added more documentation for read and write action

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

Added powers

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
added documentation for the Attribute services

delion-tholens committed:
Merge branch 'mendix:development' into opcua-connector

delion-tholens committed:
added more documentation

StephaneVanhoecke committed:
Update opc-ua-connector.md

Intro

StephaneVanhoecke committed:
Update opc-ua-connector.md

Add reference to milo
Feature chapter
Add OPC UA server version in pre requisite

StephaneVanhoecke committed:
Update opc-ua-connector.md

Re chaptering

StephaneVanhoecke committed:
Update opc-ua-connector.md

remove optional on browsing chapter title

StephaneVanhoecke committed:
Update opc-ua-connector.md

Review bullet point and nested list

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

Added part of the instructions for the monitored items

delion-tholens committed:
Update opc-ua-connector.md

Finished first draft of the Monitoring Items How to Guide

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

Table to explain binary / bit value format

StephaneVanhoecke committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/7664

delion-tholens committed:
Update opc-ua-connector.md

fixed some small typos

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
added images. added references

Luyao-Zhang-1 committed:
changed image names

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Update create-monitored-item-microflow-configuration.png

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
change indentation for todos

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
review

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

StephaneVanhoecke committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Create doc-pane.png

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/7664

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
move the image

Luyao-Zhang-1 committed:
remove images at the wrong location

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
Create browse.png

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 committed:
fix the broken link

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

delion-tholens committed:
Update opc-ua-connector.md

- minor updates on the opcua connector document

StephaneVanhoecke committed:
Update opc-ua-connector.md

remove an orphan bracket

Luyao-Zhang-1 committed:
Small style fixes

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/7664

Luyao-Zhang-1 committed:
Merge branch 'development' into pr/7664

Luyao-Zhang-1 committed:
Update opc-ua-connector.md

Luyao-Zhang-1 commented on content/en/docs/appstore/use-content/modules/opc-ua-connector.md:
```diff
@@ -0,0 +1,288 @@
+
+## Introduction
+
+[OPC](https://opcfoundation.org/) is the interoperability standard for the secure and reliable exchange of data in the industrial automation space and in other industries. You can use the [OPC-UA connector](#needs-marketplace-page-url) with your Mendix app to communicate with an OPC-UA server. The OPC-UA connector is based on [Eclipse Milo](https://github.com/eclipse/milo) client SDK, an open-sou ...
```
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
Before merging the PR, replace `#needs-marketplace-page-url` in the cross-reference with the URL of the component page in the Marketplace.
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨

Luyao-Zhang-1 reviewed (dismissed):
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
Before merging the PR, in the first paragraph, replace `#needs-marketplace-page-url` in the cross-reference with the URL of the component page in the Marketplace.
🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨

Luyao-Zhang-1 committed:
add missing files

MarkvanMents committed:
Add marketplace link

MarkvanMents reviewed (approved):
Link updated. Approving and merging.
